easybuild/easyconfigs/l/libxc/libxc-3.0.1-foss-2016b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.1'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/3.0.1/']
+
+configopts = 'FC="$F77" FCFLAGS="$FFLAGS" --enable-shared --enable-fortran'
+
+# From the libxc mailing list
+# To summarize: expect less tests to fail in libxc 2.0.2, but don't expect
+# a fully working testsuite soon (unless someone wants to volunteer to do
+# it, of course  ) In the meantime, unless the majority of the tests
+# fail, your build should be fine.
+# runtest = 'check'
```

Comment:
@pouillon We recently have consistently specified `toolchainopts` to fix broken tests, and also enabled the tests, for recent `libxc` easyconfigs, see for example #5256.

Can you sync this `libxc` (and the one in your other `libxc` PRs #5398, #5400, #5401) with the recent `libxc` easyconfigs in https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/l/libxc?

Reply:
Thanks for the quick reply. I hadn't dared enabling the tests, although they succeed on my machine. I looked both at #5256 and the HEAD of easyconfigs/develop to synchronize my files. It's a nice improvement over my previous versions.